### PR TITLE
Add script for copying upgrade-proposals to a release

### PR DESCRIPTION
**Pull request by mortalisk**

### CODE REVIEW

**Review by jondequinor — COMMENTED** [review]
If we run this on jenkins, do we need to do the git work (adding the changes, creating the commit, creating the PR) in the job directly?

I'm comparing it to https://github.com/equinor/komodo/blob/master/komodo/symlink/suggester/cli.py where the git work is done in the script itself.

**Comment by mortalisk** [comment]
If you run it on Jenkins, nothing special will happen. It will not do any git anything for anyone, no matter where it might be run.

I suggest we create small tools that can build on each other. So perhaps another script can do the git things? Perhaps we can generalize the concept of, run this script, then propose a PR of the results?

**Comment by jondequinor** [comment]
I see little value of such a script outside automation. I'd much rather just have the PR. But maybe recent RMs can chime in? @frode-aarstad

**Comment by frode-aarstad** [comment]
The automatic creation of a PR like Jonas describes is very good improvement. 

Its not much work creating the PR manually but being able to approve the PR as release manager saves some time.

File: komodo/insert_proposals.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+    pr_msg = f""":robot: Release {target} created by script
+---
+### Description
+- New Release: `{target}`
+- Based on: `{base}`
+- When: `{datetime.now()}`
+
+### Diff
+diff {base_file} {target_file}:
```
Comment (frode-aarstad) [inline komodo/insert_proposals.py:132]: Perhaps move this into the formatted block and add $ diff

File: komodo/insert_proposals.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        branch=tmp_target,
+    )
+
+    # making PR
+    base_content = repo.get_contents(base_file, ref=args.git_ref)
+    diff = diff_file_and_string(
+        b64decode(base_content.content).decode(), result, base, target
+    )
+
+    pr_msg = f""":robot: Release {target} created by script
```
Comment (frode-aarstad) [inline komodo/insert_proposals.py:124]: Perhaps remove "created by script" redundant since :robot:

**Review by frode-aarstad — APPROVED** [review]

**Pull request merged**